### ISSUE TITLE
Multiple Y-Axis in line chart

### ISSUE THREAD

**Issue by bhanushalimahesh3** [PHPOffice/PHPPresentation#352]
Is it possible to add secondary y-axis in line chart?? Can we set default color for lines. PFA. Thanks
![secondary-y-axis](https://cloud.githubusercontent.com/assets/11575628/24964176/28ba5006-1fbe-11e7-9ea7-3c335f01f97a.png)

**Comment by Progi1984** [comment]
@bhanushalimahesh3 No. Your two questions are enhancements. Added to the the current milestone :)

**Comment by bhanushalimahesh3** [comment]
@Progi1984 Thanks for quick response :). Could you help me out in which files should I make changes for seconday y axis & set default colors for lines ??.

**Comment by ajitrishabh** [comment]
@bhanushalimahesh3  have you got the secondary Y axis. if yes pls guide.

**Comment by bhanushalimahesh3** [comment]
@ajitrishabh no
